wpull/options.py

```diff
@@ -897,14 +897,19 @@ def _add_http_args(self):
 
     def _add_ssl_args(self):
         self._ssl_version_map = {
-            'auto': ssl.PROTOCOL_SSLv23,
-            'SSLv3': ssl.PROTOCOL_SSLv3,
-            'TLSv1': ssl.PROTOCOL_TLSv1,
+            # PROTOCOL_SSLv23 also selects TLS protocols
+            'auto': ssl.PROTOCOL_SSLv23
         }
 
         if hasattr(ssl, 'PROTOCOL_SSLv2'):
             self._ssl_version_map['SSLv2'] = ssl.PROTOCOL_SSLv2,
```

Comment:
I'm not sure why I have trailing commas. They shouldn't be there and the new addition with the trailing commas seems to causes the related (ignore the phantomjs ones) tests to fail.